### DISCUSSION TITLE
Slow Next.js dev server rebuild since 0.22.1 update?

### DISCUSSION THREAD

**Discussion by chrislgrigg** [chakra-ui/panda#1849] [Q&A] [answered]
I seem to be encountering extremely slow dev server reloading (2-4s but frequently > 20s) since updating to 0.22.1. Downgrading to 0.21.0 solved it immediately, reloading drops back down to 400-600ms. I tried this a few times and had the same results consistently.

Mac OS, M2 Max, Node 20.10.0, Next.js 14.0.4. I'm opening as a discussion instead of an Issue because I'm hoping to compare notes with some folks first.

Thank you!

**Comment by astahmer** [comment]
Someone on discord reported a similar issue, although not on Next.js

I'll investigate asap !

**Comment by anubra266** [comment, marked as answer]
@subvertallchris https://github.com/chakra-ui/panda/pull/1851

**Reply by chrislgrigg** [reply]
Beautiful! I'll be keeping my eye on the PR. Thank you for your work!
